halpybot/packages/case/caseutils.py

```diff
@@ -0,0 +1,104 @@
+"""
+caseutils.py - Crunching the Numbers for Case Information
+
+Copyright (c) The Hull Seals,
+All rights reserved.
+
+Licensed under the GNU General Public License
+See license.md
+"""
+from __future__ import annotations
+import re
+from typing import Dict, TYPE_CHECKING
+from pendulum import now
+from attrs import evolve
+from ..models import Case, Platform
+
+if TYPE_CHECKING:
+    from ..ircclient import HalpyBOT
+
+
+async def format_case_details(case: Case) -> str:
```

Comment:
why isn't this an instance method of `Case`.

specifically either a `__str__` or `__format__` override?

Reply:
Will move to be a class attribute of Case. 

Unsure how to proceed with "specifically either a __str__ or __format__ override"